src/eds_files/Arduino1.eds

```diff
@@ -0,0 +1,1254 @@
+[FileInfo]
```

Comment:
Welke EDS is dit? Niet van steering toch? Is dit de example van AGCON?

Reply:
Nee niet van steering dat klopt.. Ik heb deze zelf aangemaakt, wil je dat ik die van steering erin zet? Dan moet je even jouw versie opsturen. :smiley: